sdk/python/kfp/client/client.py

```diff
@@ -477,11 +477,14 @@ def create_experiment(
                 resource_references=resource_references)
             experiment = self._experiment_api.create_experiment(body=experiment)
 
+        link = f"{self._get_url_prefix()}/#/experiments/details/{experiment.id}"
         if self._is_ipython():
             import IPython
-            html = \
-                f'<a href="{self._get_url_prefix()}/#/experiments/details/{experiment.id}" target="_blank" >Experiment details</a>.'
+            html = f'<a href="{link}" target="_blank" >Experiment details</a>.'
             IPython.display.display(IPython.display.HTML(html))
+        else:
+            logging.info(f"Experiment details: {link}")
```

Comment:
If I recall correctly, default logging level is warning so info would not show up. What do you think about just using a `print()` here?

Reply:
Thanks for the feedback, @chensun! Replaced all references to `logging.info` with `print` and rebased from master.